### ISSUE TITLE
Use Indexer release for all but dev configuration

### ISSUE THREAD

**Issue by fabrice102** [algorand/sandbox#106]
## Problem

Currently, sandbox recompiles the indexer from the master branch even for the default release configuration.
This makes sandbox slower to start (compared to downloading an executable) and running a potentially non-fully stable software.

## Solution

Use the latest stable release from indexer.
This change may be simpler if the indexer is published as a Debian package.

## Urgency

Nice to have.

**Comment by fabrice102** [comment]
This seems to be related to #37

**Comment by winder** [comment]
Blocked by https://github.com/algorand/go-algorand-internal/issues/623
